addon/test/test-migrations.js

```diff
@@ -1,4 +1,4 @@
-/* global describe beforeEach it */
+/* global describe it */
```

Comment:
If you don't want to manage these `global` variables per-file, we could also set an additional `env` of `mocha: true` in the addon/.eslintrc.yml file (or even just add extra env for tests in addon/test/.eslintrc.yml).